src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -25,10 +27,267 @@ declare global {
    * ```
    */
   class Actor extends ClientDocumentMixin(foundry.documents.BaseActor) {
+    /**
+     * {@inheritdoc}
+     * @override
+     * @param data - Initial data provided to construct the Actor document
+     */
+    constructor(
+      data: ConstructorParameters<typeof foundry.documents.BaseActor>[0],
+      context: ConstructorParameters<typeof foundry.documents.BaseActor>[1]
+    );
+
+    /**
+     * An object that tracks which tracks the changes to the data model which were applied by active effects
+     * @defaultValue `{}`
+     * TODO: ActiveEffects has it as unknown @see ActiveEffect#apply
```

Comment:
What exactly is the TODO about? How should this be improved? I don't really think we can actually improve this because the values changed by effects can be of arbitrary type, right?